src/components/InstructionTable.tsx

```diff
@@ -0,0 +1,96 @@
+import { Fragment } from 'react'
+
+const instructionSet = [
+  {
+    name: 'Assertions and tests',
+    instructions: [
+      { name: 'assert', stack_input: '[a, ...]', stack_output: '[...]', cycles: '1' },
+      { name: 'assert_eq', stack_input: '[b, a, ...]', stack_output: '[...]', cycles: '1' },
+      { name: '...', stack_input: '...', stack_output: '...', cycles: '...' },
+    ],
+  },
+  // More people...
+]
+
+function classNames(...classes: string[]) {
+  return classes.filter(Boolean).join(' ')
+}
+
+export default function InstructionTable() {
+  return (
+    <div className="px-4 sm:px-6 lg:px-8">
+      <div className="sm:flex sm:items-center">
+        <div className="sm:flex-auto">
+          <p className="mt-2 text-sm text-gray-700">
+            A list of all the instructions currently possible to be used in the Miden VM.
+          </p>
+        </div>
+      </div>
+      <div className="mt-8 flex flex-col">
+        <div className="-my-2 -mx-4 overflow-x-auto sm:-mx-6 lg:-mx-8">
+          <div className="inline-block min-w-full py-2 align-middle md:px-6 lg:px-8">
+            <div className="overflow-hidden shadow ring-1 ring-black ring-opacity-5 md:rounded-lg">
+              <table className="min-w-full">
+                <thead className="bg-white">
+                  <tr>
+                    <th scope="col" className="py-3.5 pl-4 pr-3 text-left text-sm font-semibold text-gray-900 sm:pl-6">
+                      Instruction
+                    </th>
+                    <th scope="col" className="px-3 py-3.5 text-left text-sm font-semibold text-gray-900">
+                      Stack Input
+                    </th>
+                    <th scope="col" className="px-3 py-3.5 text-left text-sm font-semibold text-gray-900">
+                      Stack Output
+                    </th>
+                    <th scope="col" className="px-3 py-3.5 text-left text-sm font-semibold text-gray-900">
+                      Cycles
+                    </th>
+                    <th scope="col" className="px-3 py-3.5 text-left text-sm font-semibold text-gray-900">
```

Comment:
I would define a named for "px-3 py-3.5 text-left text-sm font-semibold text-gray-900" in `globals.css` so it doesn't need to be duplicated

Reply:
done